src/libraries/System.Text.Json/tests/Serialization/PropertyVisibilityTests.cs

```diff
@@ -456,5 +456,335 @@ public static void OverrideJsonIgnorePropertyUsingJsonPropertyNameReversed()
             string jsonSerialized = JsonSerializer.Serialize(obj);
             Assert.Equal(json, jsonSerialized);
         }
+
+        [Fact]
+        public static void JsonIgnoreConditionSetToAlwaysWorks()
+        {
+            string json = @"{""MyString"":""Random"",""MyDateTime"":""2020-03-23"",""MyInt"":4}";
+
+            var obj1 = JsonSerializer.Deserialize<ClassWithProperty_IgnoreConditionAlways>(json);
+            Assert.Equal("Random", obj1.MyString);
+            Assert.Equal(default, obj1.MyDateTime);
+            Assert.Equal(4, obj1.MyInt);
+
+            var obj2 = JsonSerializer.Deserialize<ClassWithProperty_IgnoreConditionAlways_Ctor>(json);
+            Assert.Equal("Random", obj2.MyString);
+            Assert.Equal(default, obj2.MyDateTime);
+            Assert.Equal(4, obj1.MyInt);
```

Comment:
typo: `obj1.MyInt`

Reply:
Perhaps just break this up into two tests since there is very little shared code.